src/components/TransactionList.vue

```diff
@@ -140,3 +150,885 @@ export default {
   }
 }
 </style>
+
+<docs>
+### Examples
+#### Empty
+```jsx
+  <TransactionList :transactions="[]" currency="nanoWits" />
+```
+#### With pagination
+```jsx
+  const transactions = [
```

Comment:
It's generally faster to use `JSON.parse` when creating large objects in javascript. Also it would be great if the transaction was imported from somewhere, instead of polluting the component.

Reply:
The main goal of these `docs` is show how to use a specific component. IMO, If we didn't show the props passed to the component, this `doc` would be worthless.

Reply:
On the other hand, I think the length should be reduced.